Guide/src/dev_guide/dev_tools/hypestv.md

```diff
@@ -44,11 +44,12 @@ After this, all commands will implicitly operate on `tdxvm`. Use `select` again
 to work on another VM.
 
 To enable serial port output, use the `serial` command. This can be used at any
-time, even while the VM is not running. E.g., to enable serial port output for
-COM2:
+time, even while the VM is not running. E.g., to open a separate window for
+interactive use of COM1 and enable logging serial port output for COM2:
 
 ```
-tdxvm [off]> serial 2 output
+tdxvm [off]> serial 1 term
```

Comment:
another useful addition to this sort of tool: having a `.hypestv_init` file (a-la `.gdbinit`) which lets you configure a set of pre-baked commands to run when doing various kinds of vm development (e.g: depending on the guest that's running, or the vm sku, etc...).

the concern, of course, is that this opens up an avenue to "script" this dev tool...

Reply:
I was thinking that similar to this, we'd have the concept of a workspace that could be loaded or saved by name. And maybe if you name it the same as a VM, it gets loaded automatically or something.

Reply:
something like a `.vmname` file or something like that?

Reply:
yeah, I could probably get behind a scheme like auto-loading `.hypestv-<vmname>`.